spaceros/Dockerfile

```diff
@@ -69,6 +69,7 @@ RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/r
 
 # Install required software development tools and ROS tools (and vim, included for convenience)
 RUN apt-get update && apt-get install -y \
+  apt-utils \
```

Comment:
What prompted the addition of this dependency?

Reply:
A constant warning: `debconf: delaying package configuration, since apt-utils is not installed`

Reply:
After doing some reading, I think that while this may fix the reported warning it will actually create additional problems since these utilities support the interactive apt configuration process which we cannot use during the docker build process.

apt-utils shouldn't be required with `DEBIAN_FRONTEND=noninteractive` set but I think that the message from debconf can be ignored since any packages which ultimately fail to configure should raise an error when apt-get finishes running.

Reply:
Well that checks out and makes sense, removing that dep